segpy/datatypes.py

```diff
@@ -64,9 +64,12 @@ def size_in_bytes(ctype):
 LIMITS = {
     'ibm': Limits(MIN_IBM_FLOAT, MAX_IBM_FLOAT),
     'int32': Limits(-2147483648, 2147483647),
+    'uint32': Limits(0, 4294967295),
     'int16': Limits(-32768, 32767),
+    'uint16': Limits(0, 65535),
```

Comment:
I think this should be Limits(0, 32767) because it still needs to be representable as a signed int in the file.